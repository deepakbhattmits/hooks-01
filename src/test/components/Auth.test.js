import React from "react";
import Auth from "../../components/Auth";
import { shallow } from "enzyme";
test("renders Auth correctly", () => {
    const wrapper = shallow(<Auth />);
    expect(wrapper.debug()).toMatchSnapshot();
});
