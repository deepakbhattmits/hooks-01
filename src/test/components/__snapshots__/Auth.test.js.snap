// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Auth correctly 1`] = `
"<div className=\\"auth\\">
  <Card>
    <h2>
      You are not authenticated!
    </h2>
    <p>
      Please log in to continue.
    </p>
    <button onClick={[Function: loginHandler]}>
      Log In
    </button>
  </Card>
</div>"
`;
